client/my-sites/email/email-existing-forwards-notice/index.jsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'calypso/components/notice';
+
+const EmailExistingForwardsNotice = ( { domainsWithForwards, selectedDomainName } ) => {
+	const translate = useTranslate();
+
+	if ( ! domainsWithForwards.includes( selectedDomainName ) ) {
+		return null;
+	}
+
+	return (
+		<Notice showDismiss={ false } status="is-warning">
+			{ translate(
+				"Please note that your existing email forwards for '%(domainName)s' will be removed if you upgrade to a hosted email solution. " +
+					'You will need to specify the email addresses you want to keep using below as part of the upgrade.',
```

Comment:
I think we can simplify this a bit, because the second sentence is a bit unclear to me. How about:

> Use the form below to set up the email addresses you want to continue using.